app/assets/v2/js/open-chat.js

```diff
@@ -0,0 +1,14 @@
+const openChat = () => {
+  $('[data-openchat]').each(function(index, elem) {
+    let chatWindow;
```

Comment:
do we need this variable ?

Reply:
not really now is actually so we can close the window from js if we need it

Reply:
also because Im pointing to the same window when user open a new chat.